ingestion/setup.py

```diff
@@ -115,8 +115,8 @@ def get_long_description():
     "okta": {"okta~=2.3.0"},
     "mlflow": {"mlflow-skinny~=1.22.0"},
     "sklearn": {"scikit-learn==1.0.2"},
-    "db2": {"ibm-db-sa==0.3.7"},
-    "clickhouse": {"clickhouse-driver==0.2.3", "clickhouse-sqlalchemy==0.2.0"},
+    #     "db2": {"ibm-db-sa==0.3.7"},
+    #     "clickhouse": {"clickhouse-driver==0.2.3", "clickhouse-sqlalchemy==0.2.0"},
```

Comment:
Should this be commented out? If the python CI was bringing issues, that has been fixed https://github.com/open-metadata/OpenMetadata/issues/3012

Reply:
Will uncomment this before the final review. It was for testing.